<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWidget</class>
 <widget class="QWidget" name="PlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1" native="true">
       <attribute name="title">
        <string>Data Sources</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="dataSourceTableView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="buttonSelectAll">
              <property name="text">
               <string>Select All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonDeselectAll">
              <property name="text">
               <string>Deselect All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonInvertSelection">
              <property name="text">
               <string>Invert Selection</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>278</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2" native="true">
       <attribute name="title">
        <string>Settings &amp;&amp; Actions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBoxShowLegend">
            <property name="text">
             <string>Show &amp;Legend</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxLogYAxis">
            <property name="text">
             <string>&amp;Logarithmic Y Axis</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSavePlot">
          <property name="text">
           <string>Save Plot...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>429</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="plotWrapper" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current plot.</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="text">
    <string>Deselect All</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="text">
    <string>Invert Selection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
